cmd/formula-analytics.rb

```diff
@@ -189,6 +219,24 @@ def influx_analytics(args)
           end
         when :os_versions
           format_os_version_dimension(tags["os_name_and_version"])
+        when :command_run_options
+          "#{tags["command"]} #{tags["options"]}"
+        when :test_bot_test
+          command_and_package, options = tags["command"].split.partition { |arg| !arg.start_with?("-") }
+
+          # Cleanup bad data before https://github.com/Homebrew/homebrew-test-bot/pull/1043
+          # TODO: actually delete this from InfluxDB.
+          # Can delete this code after 27th April 2025.
+          next if %w[audit install linkage style test].exclude?(command_and_package.first)
+          next if command_and_package.last.include?("/")
+          next if options.include?("--tap=")
+          next if options.include?("--only-dependencies")
+          next if options.include?("--cached")
```

Comment:
@Bo98 @SMillerDev could do with some help figuring out if we can just delete the data here. I tried and failed:

```console
$ influx delete --bucket analytics --predicate '_measurement="test_bot_test"' --start='2024-04-01T00:00:00Z' --stop='2024-04-26T07:07:19.000Z' --org-id *** --host '***' --token="***"
```

This ran successfully but didn't seem to delete anything 🤷🏻. Help?

Reply:
<img width="870" alt="Screenshot 2024-05-02 at 19 54 08" src="https://github.com/Homebrew/homebrew-formula-analytics/assets/1190754/803f6a78-e847-47be-a4f5-036fd6da1b08">

Reply:
The CLI as a whole doesn't really work with InfluxDB v3.